src/main/resources/application.properties

```diff
@@ -1 +1,9 @@
+spring.datasource.url=jdbc:h2:mem:testdb
+spring.datasource.driverClassName=org.h2.Driver
+spring.datasource.username=sa
+spring.datasource.password=password
+spring.jpa.database-platform=org.hibernate.dialect.H2Dialect
+
```

Comment:
also add
`spring.h2.console.enabled=true`
this was also mentioned in task requirements